### PR TITLE
Initialisation params changed

### PR DESCRIPTION
**Pull request by keriat**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -14,10 +15,15 @@
   "files": [
     "dist"
   ],
-  "author": "Aaron Barnard <aaron@blocknative.com>",
+  "author": {
+    "name": "Matter Labs",
+    "email": "s.beresnev@matterlabs.dev",
```
Comment (StanislavBreadless) [inline package.json:20]: When we provide the email for the projects made by Matter Labs, we use "hello@matterlabs.dev"

File: package.json [hunk 2]
```diff
@@ -38,15 +44,15 @@
...
+    "rollup-plugin-svelte": "~6.1.1",
     "rollup-plugin-typescript2": "0.21.0",
     "svelte": "^3.12.1",
     "svelte-i18n": "^1.1.2-beta",
     "typescript": "^3.6.4"
   },
   "dependencies": {
     "@ledgerhq/hw-app-eth": "^5.21.0",
-    "@ledgerhq/hw-transport-u2f": "^5.21.0",
+    "@ledgerhq/hw-transport-u2f": "^5.36.0-deprecated",
```
Comment (StanislavBreadless) [inline package.json:55]: Why use a deprecated package?

File: src/views/WalletSelect.svelte [hunk 3]
```diff
@@ -30,13 +30,23 @@
     AppState,
     WalletModule,
     WalletSelectModule,
-    WalletInterface
+    WalletInterface,
+    PopupContent
   } from '../interfaces'
 
+  const defaultPopupContent: PopupContent = {
```
Comment (StanislavBreadless) [inline src/views/WalletSelect.svelte:37]: Correct me if I'm wrong, but is seems to be the same object as in `stores.ts`

**Review by StanislavBreadless — COMMENTED** [review]
Besides minor problems, everything is great :+1:

**Comment by keriat** [comment]
Thanks, I'll update authorship and will remove duplicated code fragment.
As of the packages as we decided yesterday the maintenance for now would be limited to merging new versions from block native without changes to the used packages list to assure compatibility with original package for now.

File: src/views/WalletSelect.svelte (hunk 3 above)
Comment (keriat) [reply src/views/WalletSelect.svelte:37]: That's right, thanks for pointing out

**Pull request closed without merging**